test/libsolidity/semanticTests/underscore/as_function.sol

```diff
@@ -0,0 +1,20 @@
+contract C {
+    function _() public pure returns (uint) {
+        return 88;
+    }
+
+    function g() public pure returns (uint){
+        return _();
+    }
+
+    function h() public pure returns (uint) {
+        _;
+        return 33;
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// _() -> 88
+// g() -> 88
+// h() -> 33
```

Comment:
If these instances are going to be disallowed, does it make sense to have them as semantic tests?

Reply:
We'll see what happens when they are disallowed, right? I personally feel it is reassuring to see things break when I expect them to break and to update tests then and there.

Reply:
Ok! The syntax tests are fine though.